src/app/components/home/add-marios/add-marios.component.scss

```diff
@@ -0,0 +1,238 @@
+@import "src/styles.scss";
+$main-gray-color: #bababa;
+
+.container {
+  @include flex-display(center, center, column);
+  width: 100vw;
+  margin-bottom: 100px;
+}
+
+.form-container {
+  @include flex-display(center, flex-start, row);
+  @include font-style(700, 25px, 4px);
+  width: $content-width;
+  background-color: white;
+  margin-top: 80px;
+  border-radius: 5px;
+}
+
+.back-button-container {
+  @include flex-display(flex-start, center, row);
+  width: $content-width;
+  margin-top: 20px;
+}
+
+.ng-select.user-select {
+  width: $content-width;
+}
+
+.ng-select.user-select ::ng-deep .ng-select-container {
+  @include flex-display(center, center, row);
+  @include font-style(600, 25px, 1px);
+  border: none;
+  border-radius: 0px;
+  border-top-left-radius: 5px;
```

Comment:
I see that border-radius has the same value as in other places. Maybe create variable?